packages/backend/src/repositories/my-list.ts

```diff
@@ -9,27 +9,110 @@ export class MyListRepository {
     this._dbClient = prismaClient;
   }
 
-  public getAllUserItems({ userId }: { userId: string }) {
+  public getPurchasedItems({ userId }: { userId: string }) {
     return this._dbClient.product.findMany({
       where: {
-        OR: [
-          {
-            authorId: userId,
-            status: ProductStatus.DRAFT,
+        winnerId: userId,
+        status: ProductStatus.FINISHED,
+      },
+      select: {
```

Comment:
i've added an order entity that is needed for payment service, so i think we need added it here, when check is the product was already purchased